jobs/longitudinal_view.sh

```diff
@@ -15,3 +15,5 @@ spark-submit --executor-cores 8 \
              target/scala-2.11/telemetry-batch-view-1.1.jar \
              --to $date \
              --bucket $bucket
+
+parquet2hive -ulv 1 --sql s3://telemetry-parquet/longitudinal | beeline -u jdbc:hive2://${HIVE_SERVER}:10000
```

Comment:
This should run only if the Spark job completes succesfully. Otherwise what could happen is that the job fails,  this command runs succesfully and finally Airflow reports the whole thing as succesful.